### PR TITLE
New video server

### PR DESCRIPTION
**Pull request by nik0**
- Need paths are configurable in /usr/bin/v4l2rtspserver-master.ini
- Try server option -a to use directly audio /dev/dsp

### CODE REVIEW

File: usr/scripts/detectionOn.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# Source your custom motion configurations
+. /usr/config/motion.conf
```
Comment (jmtatsch) [inline usr/scripts/detectionOn.sh]: Can we agree on /etc/motion.conf there?

Comment (nik0) [reply usr/scripts/detectionOn.sh]: Sure ...
I was not sure how far you are on porting the scripts

File: usr/scripts/detectionOn.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		/usr/bin/telegram p "$save_dir/$filename"
+	else
+		/usr/bin/getimage > "telegram_image.jpg"
+ +	/usr/bin/telegram p "telegram_image.jpg"
+ +	rm "telegram_image.jpg"
+	fi
+fi
+
+# Run any user scripts.
+for i in /usr/config/userscripts/motiondetection/*; do
```
Comment (jmtatsch) [inline usr/scripts/detectionOn.sh]: I have userscripts in /usr/userscripts.
Not sure where user changable scripts would go on a normal linux system.
Probably /usr/local/bin ?

File: usr/bin/v4l2rtspserver-master.ini [hunk 3]
```diff
@@ -0,0 +1,29 @@
+[Configuration]
+#To save some memory you can deactivate the folowing features
+OSD=on
+JPEG=on
+MOTION=on
+
+FontFixedWidth=/usr/fonts/NotoMono-Regular.ttf
```
Comment (jmtatsch) [inline usr/bin/v4l2rtspserver-master.ini]: Can we move the fonts into /usr/share/fonts ? that would be one of the standard dirs for fonts on a linux system.

File: usr/scripts/detectionOn.sh (hunk 2 above)
Comment (nik0) [reply usr/scripts/detectionOn.sh]: Don't know either
we can choose what we want ...

File: usr/bin/v4l2rtspserver-master.ini (hunk 3 above)
Comment (nik0) [reply usr/bin/v4l2rtspserver-master.ini]: OK

File: usr/scripts/detectionOn.sh (hunk 2 above)
Comment (benleb) [reply usr/scripts/detectionOn.sh]: ```/usr/local/bin``` seems the correct place. And many users will be already familiar with it.

👍 for ```/usr/local/bin```

File: usr/bin/v4l2rtspserver-master.ini (hunk 3 above)
Comment (jmtatsch) [reply usr/bin/v4l2rtspserver-master.ini]: Perfect. Can the ini file also live in etc? and be a .conf file? for the sake of homegenity...

File: usr/scripts/detectionOff.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Source your custom motion configurations
+. /usr/config/motion.conf
```
Comment (jmtatsch) [inline usr/scripts/detectionOff.sh]: /etc/motion.conf

File: usr/scripts/detectionOff.sh [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+. /usr/scripts/common_functions.sh
+
+# Turn off the amber LED
+if [ "$motion_trigger_led" = true ] ; then
+	yellow_led off
+fi
+
+# Publish a mqtt message
+if [ "$publish_mqtt_message" = true ] ; then
+	. /usr/config/mqtt.conf
```
Comment (jmtatsch) [inline usr/scripts/detectionOff.sh]: /etc/mqtt.conf

File: usr/bin/v4l2rtspserver-master.ini (hunk 3 above)
Comment (nik0) [reply usr/bin/v4l2rtspserver-master.ini]: I didn't wanted to do that, because it more an hidden config

Comment (benleb) [reply usr/bin/v4l2rtspserver-master.ini]: Renaming *.ini Files (with ini-Format) to .conf seems strange...?

Comment (jmtatsch) [reply usr/bin/v4l2rtspserver-master.ini]: okay, then ini it stays. But shouldn't it live in the /etc directory?

File: usr/scripts/detectionOn.sh (hunk 2 above)
Comment (nik0) [reply usr/scripts/detectionOn.sh]: Go for:
for i in/usr/local/bin/motiondetection/*; do ?

Comment (jmtatsch) [reply usr/scripts/detectionOn.sh]: +1

**Pull request merged**